certbot-ci/certbot_integration_tests/rfc2136_tests/context.py

```diff
@@ -0,0 +1,70 @@
+from contextlib import contextmanager
+from pytest import skip
+from pkg_resources import resource_filename
+import tempfile
+
+from certbot_integration_tests.certbot_tests import context as certbot_context
+from certbot_integration_tests.utils import certbot_call
+
+
+class IntegrationTestsContext(certbot_context.IntegrationTestsContext):
+    """Integration test context for certbot-dns-rfc2136"""
+    def __init__(self, request):
+        super(IntegrationTestsContext, self).__init__(request)
+
+        self.request = request
+
+        self._dns_xdist = None
+
+        if hasattr(request.config, 'slaveinput'):  # Worker node
+            self._worker_id = request.config.slaveinput['slaveid']
+            self._dns_xdist = request.config.slaveinput['dns_xdist']
+        elif 'dns_xdist' in request.config:  # Primary node
+            self._worker_id = 'primary'
+            self._dns_xdist = request.config.dns_xdist
+
+    def cleanup(self):
+        super(IntegrationTestsContext, self).cleanup()
+
+    def certbot_test_rfc2136(self, args):
+        """
+        Main command to execute certbot using the RFC2136 DNS authenticator.
+        :param list args: list of arguments to pass to Certbot
+        """
+        command = ['--authenticator', 'dns-rfc2136', '--dns-rfc2136-propagation-seconds', '2']
```

Comment:
I think the response will be straightforward given that it concerns a Bind9 running locally, but do you confirm that 2 seconds for propagation is a decent choice?

Reply:
I think so, all it has to do is write to to journal file. I would be very surprised if it missed this deadline, even on overloaded CI servers.

Reply:
Works for me!